source/linux/Foundational_Components/Power_Management/pm_wakeup_sources.rst

```diff
@@ -301,26 +301,26 @@ Setup the interrupt parent and interrupt as MCU_GPIO0,
 
 .. code-block:: dts
 
-   interrupt-parent = <&mcu_gpio0>;
-   interrupts = <16 IRQ_TYPE_EDGE_RISING>;
+    interrupt-parent = <&mcu_gpio0>;
```

Comment:
https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace